### DISCUSSION TITLE
How to handle a maintenance branch from previous patch versions that already have new patch versions?

### DISCUSSION THREAD

**Discussion by Matt-Tranzact** [semantic-release/semantic-release#3431] [Q&A]
I'm getting this EINVALIDNEXTVERSION error too but when trying to make a hotfix from a patch version. e.g.
Assuming I have the following versions and tags alredy generated:
- 2.3.0
- 2.3.1
- 2.3.2
- 2.3.3
- 2.4.0

I would like to create a maintenance branch from 2.3.1 (since there is a particular client which has a problem with that version), so what i do is  `git checkout -b hotfix/2.3.1 v2.3.1`

Then I make the required change and create the commit with `fix(XXX-111): add fix for title` and push it, the github action starts to run with the semantic release step but I get the error :

> EINVALIDNEXTVERSION The release `2.3.2` on branch `hotfix/2.3.1` cannot be published as it is out of range.
> Based on the releases published on other branches, only versions within the range >=2.3.1 <2.3.2 can be published from branch hotfix/2.3.x.

Which makes kind of sense because version 2.3.2 already exists in my previous released version and tags. 

Btw, If I create the maintenance version from v.2.3.3, there are no problems during semantic release step since next version is 2.3.4. and everything is fine but I will be also adding changes from 2.3.2 and 2.3.3 which is not what I need for this scenario.

So the question would be about how to handle previous patched versions.

For this kind of scenario what would be the correct workaround? 
Is it incorrect to create a maintenance branch from a patch tag version which is previous to other ones?

**Comment by jeff-cook** [comment]
I'm running into the same issue. 

I tried the config with 
```    {
      "name": "hotfix/2.3.1",
      "prerelease": "hotfix"
    }
```

This almost did what I wanted. It creates the release `2.3.2-hotfix.1`

What I really want is `2.3.1-hotfix.1`
Which I guess is really a postrelease, not a prerelease.

**Reply by scottdickerson** [reply]
I have the exact same problem here where `"maintenance": "hotfix"` throws the invalid version since I'm trying to hotfix a preview version and `"prerelease": "hotfix"` revs the base version before appending the hotfix.1. Can we open this as a bug?  At least in the documentation

**Reply by travi** [reply]
please see my reply on the related issue: https://github.com/semantic-release/semantic-release/issues/3639#issuecomment-2719091997

also, please avoid reporting the same issue in multiple locations. we already have a hard enough time responding to all requests and that just creates more work for our maintainers
